packages/contracts-bedrock/test/cannon/MIPS2.t.sol

```diff
@@ -2463,8 +2463,210 @@ contract MIPS2_Test is CommonTest {
         assertEq(postState, outputState(expect), "unexpected post state");
     }
 
-    // TODO(client-pod#959): Port over the remaining single-threaded tests from MIPS.t.sol
-    // TODO(client-pod#959): Assert unimplemented syscalls
```

Comment:
It's already implemented in Go tests here https://github.com/ethereum-optimism/optimism/blob/a080bd23666513269ff241f1b7bc3bce74b6ad15/cannon/mipsevm/tests/evm_multithreaded_test.go#L1157.